pkg/sentry/socket/netfilter/netfilter.go

```diff
@@ -486,11 +486,11 @@ func SetEntries(stack *stack.Stack, optVal []byte) *syserr.Error {
 		// - There's some other rule after it.
 		// - There are no matchers.
 		if ruleIdx == len(table.Rules)-1 {
-			log.Warningf("Found a user chain without any rules or default policy.")
+			log.Warningf("User chain must have a rule or default policy.")
 			return syserr.ErrInvalidArgument
 		}
 		if len(table.Rules[ruleIdx].Matchers) != 0 {
-			log.Warningf("Found a user chain with matchers in first entry.")
+			log.Warningf("User chain's first node must have no matcheres.")
```

Comment:
s/matcheres/matchers/

Reply:
Oh, this was fixed when Warningf was removed. Disregard.

Reply:
Done.